test/onnx/onnx_import_dyn_shapes.in.cpp

```diff
@@ -175,3 +175,84 @@ NGRAPH_TEST(onnx_dyn_shapes_${BACKEND_NAME}, dynamic_rank_input_inference)
         test_case.run();
     }
 }
+
+NGRAPH_TEST(onnx_dyn_shapes_${BACKEND_NAME}, avg_pool_dyn_shape)
+{
+    // the model contains a single Add operation that takes a fully dynamic input and a scalar
```

Comment:
Don't know why, but still this comment is visible. Please update or remove.

Reply:
removed, done